lib/WP_Auth0_LoginManager.php

```diff
@@ -332,13 +323,11 @@ public function implicit_login() {
 		}
 
 		// Validate the nonce if one was included in the request if using auto-login.
-		if ( $this->a0_options->get( 'auto_login' ) ) {
-			$nonce = isset( $decoded_token->nonce ) ? $decoded_token->nonce : null;
-			if ( ! WP_Auth0_Nonce_Handler::getInstance()->validate( $nonce ) ) {
-				throw new WP_Auth0_LoginFlowValidationException(
-					__( 'Invalid nonce', 'wp-auth0' )
-				);
-			}
+		$nonce = isset( $decoded_token->nonce ) ? $decoded_token->nonce : null;
+		if ( ! WP_Auth0_Nonce_Handler::getInstance()->validate( $nonce ) ) {
```

Comment:
Will this work fine (fail) if the nonce was not found (null) in the id token and the flow was implicit?

Reply:
Yes. Regardless of whether we have a value or not, that `validate()` method checks against what's stored. Since `null` is not equal to whatever string we stored, this will return `false`.